Example/NYTPhotoViewer-Swift/PhotosProvider.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PhotosProvider.swift
+//  NYTPhotoViewer
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+/**
+*   In Swift 1.2, the following file level constants can be moved inside the class for better encapsulation
+*/
+let CustomEverythingPhotoIndex = 1, DefaultLoadingSpinnerPhotoIndex = 3, NoReferenceViewPhotoIndex = 4
+let PrimaryImageName = "NYTimesBuilding"
```

Comment:
In the past, The Times has generally just used string literals in code for image names, so it's a little strange to start using constants here.